src/os/TempFile.cxx

```diff
@@ -58,7 +58,7 @@ TempDir::TempDir()
 #else
     dirName_ = "./openmrntmpdirXXXXXX";
 #endif
-    dirName_.c_str();
+    (void)dirName_.c_str();
```

Comment:
Also curious on this and the reference below

Reply:
This was a warning on "unused return value".

Reply:
Interesting, Did we have a specific need to call c_str() here in the first place though?